app/controllers/admin/searchgov_domains_controller.rb

```diff
@@ -1,10 +1,13 @@
 class Admin::SearchgovDomainsController < Admin::AdminController
   active_scaffold :searchgov_domain do |config|
     config.label = 'Search.gov Domains'
-    config.actions = %i[create list search export nested]
+    config.actions = %i[create list search export nested show]
     config.create.columns = [:domain]
     config.columns = %i[id domain status urls_count unfetched_urls_count created_at]
     config.nested.add_link(:searchgov_urls, label: "URLs", page: false)
+
+    config.nested.add_link(:sitemaps, label: "Sitemaps", page: false)
```

Comment:
@ethanewing , I think you can actually remove `label: "Sitemaps"` from these arguments. I added the custom `label: "URLs"` above because I didn't want the label to appear as `Searchgov Urls`. I *believe* ActiveScaffold will automatically set the `Sitemaps` label in this case.